File: src/main/java/com/xeoneux/caravansary/CaravansaryApplication.java
package com.xeoneux.caravansary;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ComponentScan;

@SpringBootApplication
@EnableAutoConfiguration
@ComponentScan
public class CaravansaryApplication {

    public static void main(String[] args) {
        SpringApplication.run(CaravansaryApplication.class, args);
    }
}

File: src/main/kotlin/com/xeoneux/caravansary/CaravansaryApplication.kt
package com.xeoneux.caravansary

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.EnableAutoConfiguration
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.context.annotation.ComponentScan

@SpringBootApplication
@EnableAutoConfiguration
@ComponentScan
class CaravansaryApplication

fun main(args: Array<String>) {
    SpringApplication.run(CaravansaryApplication::class.java, *args)
}
